shared/wallets/receive-modal/index.js

```diff
@@ -38,66 +39,82 @@ const ReceiveModal = (props: Props) => {
         )}
-      </Box2>
+      </Kb.Box2>
       {props.isDefaultAccount && (
-        <Divider
+        <Kb.Divider
           style={{
             marginBottom: 14,
             marginTop: 10,
             width: '100%',
           }}
         />
       )}
-      <Box2 direction="vertical" fullWidth={true} style={styles.sidePaddings}>
-        <Text type="Body" style={styles.instructionText}>
+      <Kb.Box2 direction="vertical" fullWidth={true} style={styles.sidePaddings}>
+        <Kb.Text type="Body" style={styles.instructionText}>
           People outside Keybase can send to:
-        </Text>
-        <Box2 direction="vertical" gap="tiny" fullWidth={true} style={styles.stellarAddressesContainer}>
+        </Kb.Text>
+        <Kb.Box2 direction="vertical" gap="tiny" fullWidth={true} style={styles.stellarAddressesContainer}>
           <Addresses federatedAddress={props.federatedAddress} stellarAddress={props.stellarAddress} />
-          {!Styles.isMobile && <Button label="Close" onClick={props.onClose} type="Secondary" />}
-        </Box2>
-      </Box2>
+          {!Styles.isMobile && <Kb.Button label="Close" onClick={props.onClose} type="Secondary" />}
+        </Kb.Box2>
+      </Kb.Box2>
     </WalletPopup>
   )
 }
 
 const Addresses = ({federatedAddress, stellarAddress}: AddressesProps) => (
-  <Box2 direction="vertical" gap="tiny" fullWidth={true} style={styles.stellarAddressesContainer}>
+  <Kb.Box2 direction="vertical" gap="tiny" fullWidth={true} style={styles.stellarAddressesContainer}>
     {!!federatedAddress && (
-      <Box2 direction="vertical" gap="xtiny" fullWidth={true} style={styles.stellarAddressesContainer}>
-        <Text type="BodySmallSemibold">Your "federated" Stellar address:</Text>
-        <CopyText buttonType="Wallet" text={federatedAddress} />
-      </Box2>
+      <Kb.Box2 direction="vertical" gap="xtiny" fullWidth={true} style={styles.stellarAddressesContainer}>
+        <Kb.Text type="BodySmallSemibold">Your "federated" Stellar address:</Kb.Text>
+        <Kb.CopyText buttonType="Wallet" text={federatedAddress} />
+      </Kb.Box2>
     )}
-    <Box2 direction="vertical" gap="xtiny" fullWidth={true} style={styles.stellarAddressesContainer}>
-      <Text type="BodySmallSemibold">Your public Stellar address:</Text>
-      <CopyText buttonType="Wallet" text={stellarAddress} />
-    </Box2>
-  </Box2>
+    <Kb.Box2 direction="vertical" gap="xtiny" fullWidth={true} style={styles.stellarAddressesContainer}>
+      <Kb.Text type="BodySmallSemibold">Your public Stellar address:</Kb.Text>
+      <Kb.CopyText buttonType="Wallet" text={stellarAddress} />
+    </Kb.Box2>
+    <Kb.Box2 direction="vertical" gap="xtiny" fullWidth={true} style={styles.stellarAddressesContainer}>
+      <Kb.Text type="BodySmallSemibold">Your Stellar QR code:</Kb.Text>
+      <Kb.Box2 direction="vertical" fullWidth={true} style={styles.qrContainer} centerChildren={true}>
+        <QrImage address={stellarAddress} />
```

Comment:
If this is the default account we should use the federated address here so this ends up as a direct payment